src/traces/heatmap/calc.js

```diff
@@ -82,21 +83,21 @@ module.exports = function calc(gd, trace) {
 
     function noZsmooth(msg) {
         zsmooth = trace._input.zsmooth = trace.zsmooth = false;
-        Lib.notifier('cannot fast-zsmooth: ' + msg);
+        Lib.notifier(_(gd, 'cannot use zsmooth: "fast"') + '<br>' + msg);
```

Comment:
Do we really want this (and the 3 msgs below) to be notified? Can I just turn it into a console warning and leave it in English?

Reply:
> Can I just turn it into a console warning and leave it in English?

I vote :+1:

Reply:
I think `Lib.notifier` should only be used for things that might affect the end user. 

`Lib.{log,warn,error}` should suffice for messages that only help developers debug .

Reply:
moved these to `Lib.warn` and turned log level 1 on by default in https://github.com/plotly/plotly.js/pull/2195/commits/0f8520e846f4251b477e265fb12cfd4fa1d41509

see also https://github.com/plotly/plotly.js/issues/420#issuecomment-350083770